Source/Debugger.pas

```diff
@@ -991,8 +1003,15 @@ procedure TDebugger.LogStackFrame(const ADebugEvent: DEBUG_EVENT);
   DebugThread: IDebugThread;
   Module: IDebugModule;
   MapScanner: TJCLMapScanner;
+  MachineType: Cardinal;
 begin
   ContextRecord.ContextFlags := CONTEXT_ALL;
+  case FProcessTarget of
+    taWin32:
+      MachineType := IMAGE_FILE_MACHINE_I386;
+    taWin64:
+      MachineType := IMAGE_FILE_MACHINE_AMD64;

```

Comment:
This should have an else branch or compiler shows warning. Maybe 
  else
  begin
      FLogManager.Log('Unkown platform');
      Exit;
  end;

Reply:
Yeah, I was lazy, sorry my bad, solved.